libs/wingsdk/src/cloud/function.ts

```diff
@@ -144,7 +144,7 @@ export abstract class Function extends Resource implements IInflightHost {
  */
 export interface IFunctionClient {
   /**
-   * Invoke the function asynchronously with a given payload.
+   * Invokes the function synchronously with a payload and waits for the result.
```

Comment:
I'm not sure it's worth mentioning async vs sync here. The function itself is not blocking, but that's implied by the return type.

```suggestion
   * Invokes the function with a payload and waits for the result.
```

Reply:
the difference between this and `invokeAsync` is less about the async-ness of the function and more about the "waits for the result" part